Makefile

```diff
@@ -72,7 +73,7 @@ build/pyodide.asm.js: src/main.bc src/type_conversion/jsimport.bc \
 	date +"[%F %T] Building pyodide.asm.js..."
 	[ -d build ] || mkdir build
 	$(CXX) -s EXPORT_NAME="'pyodide'" -o build/pyodide.asm.js $(filter %.bc,$^) \
-		$(LDFLAGS) -s FORCE_FILESYSTEM=1 --preload-file root/lib@lib
+		$(LDFLAGS) $(EXTRA_LDFLAGS) -s FORCE_FILESYSTEM=1 --preload-file root/lib@lib
```

Comment:
EXTRA_LDFLAGS doesn't appear to be defined

Reply:
These two are yet again in the "Hood isn't all that great at git category": I didn't want to commit local changes to Makefile, but I said `git commit -a`.